kbase/narrative
# DATAUP-312 write Python code for running

# Description of PR purpose/changes

This includes a few minor fixes, the generated Python code for running the cell, and toggles the action button to run when ready.

# Jira Ticket / Issue #
https://kbase-jira.atlassian.net/browse/DATAUP-312

# Testing Instructions
* Details for how to test the PR:
1. You might need to make a new bulk import cell with these changes
2. Fill out all parameters and file paths
3. It should now show check marks in both file type buttons, and make the Run button available to click.
4. Clicking that run button should put a Pythonic error at the bottom of the cell, as the `run_app_bulk` function isn't written in this PR.

# Dev Checklist:
